Sources/Moya/Plugins/AccessTokenPlugin.swift

```diff
@@ -6,15 +6,13 @@ import Foundation
 public protocol AccessTokenAuthorizable {
 
     /// Represents the authorization header to use for requests.
-    var authorizationType: AuthorizationType { get }
+    var authorizationType: AuthorizationType? { get }
```

Comment:
Hey @amaurydavid 
What's the advantage in making this optional? I mean instead of using `.none`. I'm asking because at the end using AuthorizationType optional nil a.k.a Optional<AuthorizationType>.none will be almost the same that  `AuthorizationType.none`

Reply:
The reasons are in the PR description ;)

Reply:
Oops 😅